apps/create-vesting/src/utils/schema.ts

```diff
@@ -3,34 +3,32 @@ import { z } from "zod";
 
 export const periodScaleOptions = ["Day", "Week", "Month"];
 
-const refinePeriodScale = (value: string) => {
-  return periodScaleOptions.includes(value);
+const isValidRecipient = (recipient: string) => {
+  // better ENS validation is performed on form submit
```

Comment:
Why to perform another validation on form submit? 🤔 Think we should stick with the pattern of doing all validations here.

Reply:
The problem is that we cant use the ENS name for the hooks, so we need to process the ens name to become an address in the app page. At first, there were two fetches to check ens, one in the zod validation, other to get the address in the page. To avoid fetching the same data twice, I removed his one from zod